PhotonTheme.Shared/Controls/Fields/VerificationBox.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using PhotonTheme.Shared.Controls.Enums;
+
+namespace PhotonTheme.Shared.Controls.Fields
+{
+    public sealed class VerificationBox : TextBox, IVerificationField
+    {
+
+        protected override void OnTextChanged(TextChangedEventArgs e)
+        {
+            base.OnTextChanged(e);
+            if (VerificationFunction != null) Verify();
+        }
+
+        #region IsError
+
+        public bool IsError
+        {
+            get => (bool)GetValue(IsErrorProperty);
+            set => SetValue(IsErrorProperty, value);
+        }
+
+        public static readonly DependencyProperty IsErrorProperty =
+            DependencyProperty.Register("IsError", typeof(bool), typeof(VerificationBox), new PropertyMetadata(false));
```

Comment:
Use better nameof(IsError) instead "IsError". It looks better and is easier to refactor.

Formatting: 

```csharp
        public static readonly DependencyProperty IsErrorProperty =
            DependencyProperty.Register(
                nameof(...),
                typeof(bool),
                typeof(...),
                new PropertyMetadata(false)
            );
```

Reply:
I do agree. I tend to overuse snippets and it easy to miss this. Thanks.